src/parser.rs

```diff
@@ -7710,6 +7720,22 @@ mod tests {
         );
     }
 
+    #[test]
+    fn test_unnest() {
```

Comment:
Can you please move these tests to the other parser tests in https://github.com/sqlparser-rs/sqlparser-rs/blob/efd8cb7fd1c4e6654b101868f7a76d80f66da74b/tests/sqlparser_common.rs#L1361-L1380

Using `one_statement_parses_to` might be good to verify the round trip. Something like

https://github.com/sqlparser-rs/sqlparser-rs/blob/efd8cb7fd1c4e6654b101868f7a76d80f66da74b/tests/sqlparser_common.rs#L1805-L1808

Reply:
Unnest does not support for all dialect so I cant test with `one_statement_parses_to`.
Can we change remove #L5998 to support all dialect?

https://github.com/sqlparser-rs/sqlparser-rs/blob/f05f71e20dc54f14cac80a638e2ed252b6fca91c/src/parser.rs#L5998-L5999

Reply:
I can't easily add a test with `one_statement_parses_to`.
I got the error `Parse results with GenericDialect are different from PostgreSqlDialect`.

Reply:
I don't understand this comment -- I believe the PR looks really nice and well tested now. Let me know if I missed anything

Reply:
I just want to show that these two test cases failed with `one_statement_parses_to`, so I did not add them to the test. I am not sure if these are expected behavior.